backend/src/middleware/handleNotifications/handleNotificationsMiddleware.js

```diff
@@ -0,0 +1,156 @@
+import { UserInputError } from 'apollo-server'
+import extractMentionedUsers from './notifications/extractMentionedUsers'
+import extractHashtags from './hashtags/extractHashtags'
+
+const notifyUsers = async (label, id, idsOfUsers, reason, context) => {
+  if (!idsOfUsers.length) return
+
+  // Done here, because Neode validation is not working.
```

Comment:
I think the neode validations only work if you use neode
Like, if you use it in the resolver than it should work, but I think we are not, so it doesn't work.
If you create a test using neode and have defined that an attribute is required and don't use it, the validations should work.

Reply:
I changed the comment to
`// Checked here, because it does not go through GraphQL checks at all in this file.`
@mattwr18 

Or